frappe/desk/doctype/notification_log/notification_log.json

```diff
@@ -91,6 +91,12 @@
    "fieldname": "attachment_link",
    "fieldtype": "HTML",
    "label": "Attachment Link"
+  },
+  {
+   "fieldname": "custom_uri",
+   "fieldtype": "Data",
+   "hidden": 1,
+   "label": "Custom Uri"
```

Comment:
Better name IMO: `link` or `url`. `custom_` prefix isn't required. 

We can change the old `document link` label to `Document Name` to correctly reflect what it is.